### DISCUSSION TITLE
Cannot run `gatsby develop` on Mac M1

### DISCUSSION THREAD

**Discussion by Cecile-Lebleu** [gatsbyjs/gatsby#29891] [Help]
Hi!

I'm having an impossible time running `gatsby develop` on a Mac Mini M1. I have been looking everywhere and asked on SO, but haven't found help or information anywhere regarding this specific issue.

When I run `gatsby develop`, I get the following:
```
$ gatsby develop
success open and validate gatsby-configs - 0.063s
⠋ load plugins
[1]    92754 killed     gatsby develop
```
And that's it. Unfortunately there's no error or information on why it's not working, so my question on SO was downvoted.
No idea what 92754 means in this case.

`gatsby build` runs just fine, but I have no idea why develop doesn't work, and there are no errors or logs to go forward from. I genuinely have no idea what's happening and haven't found help or information anywhere.

Here's the SO question (nothing there so far): https://stackoverflow.com/questions/66374246/cant-run-gatsby-develop-on-existing-project-on-mac-m1

Thank you in advance for any help, it will be very much appreciated. 
If you need any other information, I'll be happy to provide it.

Node: v15.8.0
npm: 7.5.4
Gatsby CLI version: 2.19.1
Gatsby version: 2.32.8

**Comment by Cecile-Lebleu** [comment]
In case it helps anyone, prior to this error (where it just doesn't run, without throwing any errors or clues) I had a hard time with `npm install`, but I was able to fix that by running the following:

```
brew reinstall vips
xcode-select --install
brew install gcc
brew reinstall vips
brew info vips
npm i
```

**Reply by dwarkeshsp** [reply]
This fixed the issue for me! Thank you so much!

**Reply by sriramgroot** [reply]
@Cecile-Lebleu thanks for the solution it worked for me.

**Reply by tijsluitse** [reply]
Worked! Thanks!

**Comment by ceosss** [comment]
@Cecile-Lebleu ,
Still not working for me 😢

**Comment by Cecile-Lebleu** [comment]
@ceosss Unfortunately it's still not working for me either.
After talking to Gatsby support, countless emails back and forth, following every question to try to make this work, it still doesn't.

I'm done with Gatsby, haven't used it for any new projects ever since moving to M1. But I still have to manage, update, and change the websites I built back when I thought Gatsby was the best thing to ever happen to static web. I have to keep my old laptop around for the few projects I need to keep working on, but it's so slow, so incredibly frustrating, and demoralizing to try everything and just get nowhere with this. It's great for new projects, but if it just stops working altogether just a year after creating it, it's not a tool I can trust. I'm really sorry for ranting on about this, but I see new features and versions coming out, while a problem as fundamental as this gets forgotten.

The only solution to this I see is to spend days restarting the project all over again, which sucks.

**Reply by KyleAMathews** [reply]
Have you tried uninstalling vips? If you have vips installed then sharp thinks it need to compile itself from scratch which requires a lot more things to be installed on your machine. If you uninstall vips then sharp will download a pre-built lib for your machine which is a lot faster/easier.

**Reply by LekoArts** [reply]
Since v0.28 of sharp the manual installation of vips shouldn't be necessary anymore: https://github.com/lovell/sharp/issues/2460#issuecomment-809499444

**Reply by Cecile-Lebleu** [reply]
@KyleAMathews @LekoArts Thank you very much, it finally works.
Uninstalling vips did fix it for all other projects, but for this one in particular I also had to remove it and manually update packages one by one until I found the exact combination of versions that allowed installing. A couple of plugins weren't compatible and had to be removed (react-magic-grid and some others I can't remember).
I apologize for my behavior and comments from before, frustration just doesn't justify it. Thank you for making Gatsby such an awesome product, you guys rock.

**Reply by LekoArts** [reply]
No worries, happy to hear it's working now :)

**Reply by kartikpuri95** [reply]
Uninstalling vips and then doing npm install worked for me 
Thanks @KyleAMathews

**Reply by mbappai** [reply]
> Since v0.28 of sharp the manual installation of vips shouldn't be necessary anymore: [lovell/sharp#2460 (comment)](https://github.com/lovell/sharp/issues/2460#issuecomment-809499444)

I'm using version 0.32 and it doesn't seem to be doing the trick

**Comment by pmnord** [comment]
I had this same issue and resolved it by deleting my pnpm store at `/Users/{username}/Library/pnpm/store`.
